WIPv73Changelog.md

```diff
@@ -0,0 +1,799 @@
+    | PaymentIntent | `PaymentIntentNextActionWechatPayDisplayQRCode` | `PaymentIntentNextActionWeChatPayDisplayQRCode` |
+    | PaymentIntent | `PaymentIntentNextActionWechatPayRedirectToAndroidApp` | `PaymentIntentNextActionWeChatPayRedirectToAndroidApp` |
+    | PaymentIntent | `PaymentIntentNextActionWechatPayRedirectToIOSApp` | `PaymentIntentNextActionWeChatPayRedirectToIOSApp` |
+    | PaymentIntent | `PaymentIntentNextAction.WechatPayDisplayQRCode` | `PaymentIntentNextAction.WeChatPayDisplayQRCode` |
+    | PaymentIntent | `PaymentIntentNextAction.WechatPayRedirectToAndroidApp` | `PaymentIntentNextAction.WeChatPayRedirectToAndroidApp` |
+    | PaymentIntent | `PaymentIntentNextAction.WechatPayRedirectToIOSApp` | `PaymentIntentNextAction.WeChatPayRedirectToIOSApp` |
+    | PaymentIntent | `PaymentIntentPaymentMethodOptionsWechatPay` | `PaymentIntentPaymentMethodOptionsWeChatPay` |
+    | PaymentIntent | `PaymentIntentPaymentMethodOptions.WeChatPay` | `PaymentIntentPaymentMethodOptions.WeChatPay` |
+    | PaymentLink | `PaymentLinkPaymentMethodTypeWechatPay` | `PaymentLinkPaymentMethodTypeWeChatPay` |
+    | PaymentMethod | `PaymentMethodTypeWechatPay` | `PaymentMethodTypeWeChatPay` |
+    | PaymentMethod | `PaymentMethodWechatPayParams` | `PaymentMethodWeChatPayParams` |
+    | PaymentMethod | `PaymentMethodParams.WeChatPay` | `PaymentMethodParams.WeChatPay` |
+    | PaymentMethod | `PaymentMethodWechatPay` | `PaymentMethodWeChatPay` |
+    | PaymentMethod | `PaymentMethod.WeChatPay` | `PaymentMethod.WeChatPay` |
+    | SetupIntent | `SetupIntentPaymentMethodDataWechatPayParams` | `SetupIntentPaymentMethodDataWeChatPayParams` |
+    | SetupIntent | `SetupIntentPaymentMethodDataParams.WechatPay` | `SetupIntentPaymentMethodDataParams.WeChatPay` |
+    | SetupIntent | `SetupIntentConfirmPaymentMethodDataWechatPayParams` | `SetupIntentConfirmPaymentMethodDataWeChatPayParams` |
+    | SetupIntent | `SetupIntentConfirmPaymentMethodDataParams.WechatPay` | `SetupIntentConfirmPaymentMethodDataParams.WeChatPay` |
+    | Source | `SourceWechat` | `SourceWeChat` |
+    | Source | `Source.Wechat` | `Source.WeChat` |
+    | Subscription | `SubscriptionPaymentSettingsPaymentMethodTypeWechatPay` | `SubscriptionPaymentSettingsPaymentMethodTypeWeChatPay` |
+
+  - `ExternalAccount` to `AccountExternalAccount`
+    | Resource | Old name | New name |
+    | --- | --- | --- |
+    | Account | `ExternalAccount` | `AccountExternalAccount` |
+    | Account | `ExternalAccountType` | `AccountExternalAccountType` |
+    | Account | `ExternalAccountTypeBankAccount` | `AccountExternalAccountTypeBankAccount` |
+    | Account | `ExternalAccountTypeCard` | `AccountExternalAccountTypeCard` |
+    | Account | `AccountExternalAccountParams` | `AccountExternalAccountParams` |
+    | Account | `ExternalAccountList` | `AccountExternalAccountList` | <!-- ---- R, S -->
```

Comment:
What are the `<!-- ---- R, S -->` ?

Reply:
This was to show where to merge in Kamil's stuff (for alphabetizing). Will remove